airflow/models/dagbag.py

```diff
@@ -409,11 +409,15 @@ def collect_dags(
             stats, key=lambda x: x.duration, reverse=True)
         for file_stat in self.dagbag_stats:
             dag_ids = dags_by_name[file_stat.dags]
-            if file_stat.dag_num >= 1:
-                # if we found multiple dags per file, the stat is 'dag_id1 _ dag_id2'
-                dag_names = '_'.join(dag_ids)
+            if file_stat.dag_num == 1:
+                # if the file has only one dag, it uses dag.loading-duration.dagID for metric
                 Stats.timing('dag.loading-duration.{}'.
-                             format(dag_names),
+                             format(dag_ids[0]),
+                             file_stat.duration)
```

Comment:
I think for consistency we should always use the filename - otherwise any monitoring dashboards of load time will be "mixed"

Reply:
I agree

Reply:
make sense, will do.